src/express/utils.js

```
@@ -22,6 +23,25 @@
 const STACK_ITEM_VALID_NAMES = ["router", "bound dispatch", "mounted_app"];
 
+/**
+ * Detects Express version and returns router information
+ * @param {import('express').Express | import('express').Router | any} app
+ * @returns {{stack: any[] | null, version: 'v4' | 'v5'}}
+ */
+export const getRouterInfo = function (app) {
+  if (app.stack) {
+    // Express 4 router
+    return { stack: app.stack, version: "v4" };
+  } else if (app._router && app._router.stack) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)